billing.md

```diff
@@ -99,34 +95,212 @@ Next, add the `Billable` trait to your model definition. This trait provides var
+Next, you should configure your Stripe key in your `.env` file. You can retrieve your Stripe API keys from the Stripe control panel.
 
-    'stripe' => [
-        'model' => App\User::class,
-        'key' => env('STRIPE_KEY'),
-        'secret' => env('STRIPE_SECRET'),
-        'webhook' => [
-            'secret' => env('STRIPE_WEBHOOK_SECRET'),
-            'tolerance' => env('STRIPE_WEBHOOK_TOLERANCE', 300),
-        ],
-    ],
+    STRIPE_KEY=your-stripe-key
+    STRIPE_SECRET=your-stripe-secret
 
 <a name="currency-configuration"></a>
 ### Currency Configuration
 
-The default Cashier currency is United States Dollars (USD). You can change the default currency by calling the `Cashier::useCurrency` method from within the `boot` method of one of your service providers. The `useCurrency` method accepts two string parameters: the currency and the currency's symbol:
+The default Cashier currency is United States Dollars (USD). You can change the default currency by setting the `CASHIER_CURRENCY` environment variable:
 
-    use Laravel\Cashier\Cashier;
+    CASHIER_CURRENCY=EUR
+
+In addition to configuring Cashier's currency, you may also specify a locale to be used when formatting money values for display on invoices. Internally, Cashier utilizes [PHP's `NumberFormatter` class](https://www.php.net/manual/en/class.numberformatter.php) to set the currency locale:
+
+    CASHIER_CURRENCY_LOCALE=nl_BE
+
+> {note} In order to use locales other than `en`, ensure the `ext-intl` PHP extension is installed and configured on your server.
+
+<a name="customers"></a>
+## Customers
+
+<a name="creating-customers"></a>
+### Creating Customers
+
+Occasionally, you may wish to create a Stripe customer without beginning a subscription. You may accomplish this using the `createAsStripeCustomer` method:
+
+    $user->createAsStripeCustomer();
+
+Once the customer has been created in Stripe, you may begin a subscription at a later date.
+
+<a name="payment-methods"></a>
+## Payment Methods
+
+<a name="storing-payment-methods"></a>
+### Storing Payment Methods
+
+In order to create subscriptions or perform "one off" charges with Stripe, you will need to store a payment method and retrieve its identifier from Stripe. The approach used to accomplish differs based on whether you plan to use the payment method for subscriptions or single charges, so we will examine both below.
+
+#### Payment Methods For Subscriptions
+
+When storing credit cards to a customer for future use, the Stripe Setup Intents API to must be used to securely gather the customer's payment method details. A "Setup Intent" indicates to Stripe the intention to charge a customer's payment method. Cashier's `Billable` trait includes the `createSetupIntent` to easily create a new Setup Intent. You should call this method from the route or controller that will render the form which gathers your customer's payment method details:
```

Comment:
There's a typo here: `the Stripe Setup Intents API *to must* be used to securely`